src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -217,4 +218,67 @@ privacy_notices:
   has_gpc_flag: True
   displayed_in_privacy_center: True
   displayed_in_overlay: True
+  displayed_in_api: False
+- name: Targeted Advertising
+  notice_key: targeted_advertising
+  id: pri_0cde6efc-efe9-402c-ab82-5c001e5b6tar
+  description: We may use some of your information and share it with partners for the purpose of targeted advertising. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Targeted advertising" means displaying advertisements to a consumer where the advertisement is selected based on personal data obtained or inferred from that consumer's activities over time and across nonaffiliated Internet web sites or online applications to predict such consumer's preferences or interests.
+  regions:
+    - ca_ab
+    - ca_bc
+    - ca_mb
+    - ca_nb
+    - ca_nl
+    - ca_ns
+    - ca_on
+    - ca_pe
+    - ca_sk
+    - ca_nt
+    - ca_nu
+    - ca_yt
+  consent_mechanism: opt_out
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: frontend
+  disabled: False
+  has_gpc_flag: True
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Targeted Advertising and Marketing
+  notice_key: targeted_advertising_marketing
+  id: pri_c124c60f-b278-4ae6-a6f2-9b086b763tar
+  description: We may use some of your information and share it with partners for the purpose of marketing tracking and targeted advertising. You can learn more about what information is used for this purpose in our privacy notice.
```

Comment:
just clarifying that you meant "marketing tracking and targeted advertising", that "marketing tracking" is a thing, or you meant "marketing, tracking, and targeted advertising"

Reply:
Hmmm.. weird. Should have commas. I'll update